rocAL/rocAL/include/api/rocal_api_meta_data.h

```diff
@@ -176,7 +188,7 @@ extern "C" void ROCAL_API_CALL rocalBoxEncoder(RocalContext p_context, std::vect
 
 /// \param boxes_buf  user's buffer that will be filled with encoded bounding boxes . Its needs to be at least of size batch_size.
 /// \param labels_buf  user's buffer that will be filled with encoded labels . Its needs to be at least of size batch_size.
-extern "C" void ROCAL_API_CALL rocalCopyEncodedBoxesAndLables(RocalContext p_context, float* boxes_buf, int* labels_buf);
+extern "C" void ROCAL_API_CALL rocalCopyEncodedBoxesAndLables(RocalContext p_context, double* boxes_buf, int* labels_buf);
```

Comment:
Does the training uses encoded boxes and labels in double? Otherwise we should return in float. We can do the internal calc in double, but keep the data at the end in float

Reply:
The mask training doesn't use this API or box encoder. Changed the dtype of the API parameter since the bboxes dtype is changed to double in metadata.h